src/jdk.incubator.foreign/share/classes/jdk/incubator/foreign/CLinker.java

```diff
@@ -323,13 +321,30 @@ static MemorySegment toCString(String str, Charset charset) {
      * control over the encoding process is required.
      *
      * @param str the Java string to be converted into a C string.
-     * @param scope the scope to be used for the native segment allocation.
+     * @param allocator the scope to be used for the native segment allocation.
```

Comment:
```suggestion
     * @param allocator the allocator to be used for the native segment allocation.
```

See also use of scope above at line 315. Similarly for other methods.